code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -418,6 +418,15 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	var/obj/item/organ/external/hand/l_hand = get_limb_by_name("l_hand")
 	var/obj/item/organ/external/hand/right/r_hand = get_limb_by_name("r_hand")
 
+	if(isgolem(src))
```

Comment:
scope this to 
`/mob/living/carbon/human/golem/update_hands_layer()`

Reply:
Scoped

_I think_